app/scripts/controllers/plugins.js

```diff
@@ -21,6 +30,12 @@ const SES = (
     : require('ses')
 )
 
+const getGetDomainMetadataFunction = (pluginName) => {
```

Comment:
`getGet` looks like a typo.

Reply:
I guess it isn't? I guess this really is a getter for a getter function, so maybe that's fine, I wouldn't block over it, just saying.